eclide/mainfrm.cpp

```diff
@@ -1057,6 +1075,17 @@ void CMainFrame::InitializeRibbon()
         pPanelLocate->Add(new CMFCRibbonButton(ID_EDIT_REPLACE, _T("Replace\ne"), 5));
         pButtFind->AddSubItem(new CMFCRibbonButton(ID_EDIT_GOTO, _T("Goto\ng"), 30));
 
+        CMFCRibbonPanel* pPanelBookmark = m_CategoryHome->AddPanel(_T("Bookmarks"), m_PanelImages.ExtractIcon(27));
+        CMFCRibbonButton * pButtBook = new CMFCRibbonButton(ID_BOOKMARKS_VIEW, _T("View\nf"), 10, 9);
+        pPanelBookmark->Add(pButtBook);
+        pPanelBookmark->Add(new CMFCRibbonButton(ID_BOOKMARKS_NEXT, _T("Next\no"), 6));
+        pPanelBookmark->Add(new CMFCRibbonButton(ID_BOOKMARKS_PREV, _T("Previous\ns"), 7));
+        pPanelBookmark->Add(new CMFCRibbonButton(ID_BOOKMARKS_CLEAR, _T("Clear\nl")));
+        pButtBook->AddSubItem(new CMFCRibbonButton(ID_BOOKMARKS_OPEN, _T("Open\ns")));
+        pButtBook->AddSubItem(new CMFCRibbonButton(ID_BOOKMARKS_SAVE, _T("Save\ns")));
+        pButtBook->AddSubItem(new CMFCRibbonButton(ID_BOOKMARKS_LOAD, _T("Load\nl")));
+        pButtBook->AddSubItem(new CMFCRibbonButton(ID_BOOKMARKS_DELETE, _T("Delete\ns")));
```

Comment:
Your using a lot of `s` for the keyboard shortcuts...

Reply:
Ditto...